docs/_docs/installation.md

```diff
@@ -15,9 +15,9 @@ Before you start, make sure your system has the following:
 
 - GNU/Linux, Unix, or macOS
 - [Ruby](https://www.ruby-lang.org/en/downloads/) version 2.2.5 or above, including all development
-  headers
-- [RubyGems](https://rubygems.org/pages/download)
-- [GCC](https://gcc.gnu.org/install/) and [Make](https://www.gnu.org/software/make/) (in case your system doesn't have them installed, which you can check by running `gcc -v` and `make -v` in your system's command line interface)
+  headers (ruby installation can check by running `ruby -v`, development headers can be checked on Ubuntu by running `apt list --installed  ruby-dev`)
+- [RubyGems](https://rubygems.org/pages/download) (which you can check by running `gem -v`)
+- [GCC](https://gcc.gnu.org/install/) and [Make](https://www.gnu.org/software/make/) (in case your system doesn't have them installed, which you can check by running `gcc -v`,`g++ -v`  and `make -v` in your system's command line interface)
```

Comment:
Is `g++` necessary?

Reply:
Yes, otherwise I catch following error: 
![jekyll](https://user-images.githubusercontent.com/11160215/34914241-b6e6d396-f91f-11e7-8321-fd7f7aae27c4.png)